test/TEST-55-OOMD/test.sh

```diff
@@ -2,10 +2,23 @@
 set -e
 
 TEST_DESCRIPTION="systemd-oomd Memory Pressure Test"
+IMAGE_NAME="oomd"
 
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
 
+test_append_files() {
+    # Create a swap device
+    (
+        mkswap "${LOOPDEV:?}p2"
```

Comment:
how come p2 just exists? i.e. why no change that adds in the partition first? /me is puzzled

Reply:
Our "framework" creates a two-partition FS on the lodev for each test:

https://github.com/systemd/systemd/blob/3a1e9d8083b83f611a23cc84ba69a8175e659629/test/test-functions#L987-L997